<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>535</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="instructionRateButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="TestLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Microbenchmarks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="memBandwidthButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Bandwidth</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memLatencyButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Latency</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coherencyLatencyButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Coherency Latency</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="testOutput">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="hugePagesButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Huge Pages</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryLatencyOptions</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="numaButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>64</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numa</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryLatencyOptions</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="autoNumaButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Auto Numa</string>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryLatencyOptions</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="memLatencyDefaultButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryLatencyOptions</string>
    </attribute>
   </widget>
   <widget class="QSpinBox" name="memBandwidthThreadsBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>64</number>
    </property>
   </widget>
   <widget class="QLabel" name="memBandwidthThreadsLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>41</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threads</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="memBandwidthPrivateButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Private</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryBandwidthOptions</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="memBandwidthSharedButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Shared</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">MemoryBandwidthOptions</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="CoherencyIterationsLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iterations</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="coherencyLatencyIterationsBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000000</number>
    </property>
    <property name="maximum">
     <number>50000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="stopTestButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearOutputButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="MemoryBandwidthOptions"/>
  <buttongroup name="MemoryLatencyOptions"/>
 </buttongroups>
</ui>
